docs/package.json

```diff
@@ -10,9 +10,9 @@
     "astro": "astro"
   },
   "dependencies": {
-    "@astrojs/starlight": "^0.30.1",
-    "@interledger/docs-design-system": "^0.5.5",
-    "astro": "5.0.5",
+    "@astrojs/starlight": "^0.30.2",
+    "@interledger/docs-design-system": "^0.6.0",
```

Comment:
Needs the latest dds because the new version implements the ability to pass a class name to the CodeBlock component.